package.json

```diff
@@ -52,8 +52,8 @@
     "mathjs": "^5.10.3",
     "mkdir": "^0.0.2",
     "moment-timezone": "^0.5.43",
-    "mongodb": "^3.7.4",
-    "mongoose": "^7.4.5",
+    "mongodb": "^6.8.0",
```

Comment:
@borkarsaish65 any particular reason for package updates?

Reply:
i was getting a persistent error which was a major blocker for me. It was related to bson error not letting me proceed with the code unless I upgraded the packages.